src/components/index.js

```diff
@@ -5,8 +5,8 @@ export Jumbotron from './Jumbotron';
 
 // Elements
 export Tabs from './Tabs';
-export Pane from './Tabs/Pane';
-export TabsPane from './Tabs/Pane'; // for backwards compatibility
+export TabsPane from './Tabs/Pane';
+export ImagePicker from './ImagePicker';
```

Comment:
I think this should have been an addition rather than a replacement.

As it stands, this change will break the document conversion demo because it imports `Pane` rather than `TabsPane`